pkgs/by-name/go/gomanagedocker/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+  xorg,
+}:
+
+buildGoModule rec {
+  pname = "gomanagedocker";
+  version = "1.4";
+
+  src = fetchFromGitHub {
+    owner = "ajayd-san";
+    repo = "gomanagedocker";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
This should fix the error above.